### DISCUSSION TITLE
Can I report a bug in Firefox on Github?

### DISCUSSION THREAD

**Discussion by WCarp** [community/community#111066] [New to GitHub] [answered]
I am having a terrible time trying to get Firefox working after a problem occurred (lost some Bookmarks in a folder). I cannot restore all of my Bookmarks now which are vital, especially since I do a lot of research. I started by contacting Firefox Support on 3/1/2024 and there have been some responses, some taking a while but their Support has been unable to resolve the issue of running Firefox with a valid Bookmark backup according to the “Firefox Bookmark Backup Reader/Decompressor”, including an HTML file as of today 3/5/2024.

I really need to get Firefox functional again soon! How can I get this done? I'm beginning to think there must be a bug in Firefox since Mozilla's Firefox Support is NOT resolving it.

Thank you.

**Comment by koeppel** [comment, marked as answer]
Hi,

according to the official Mozilla Firefox page you can file a bug here: https://bugzilla.mozilla.org/home
